services/app/lib/codebattle_web/controllers/api/user_controller.ex

```diff
@@ -0,0 +1,10 @@
+defmodule CodebattleWeb.Api.V1.UserController do
+  use CodebattleWeb, :controller
+
+  alias Codebattle.{User.Stats}
+
```

Comment:
If you want to use `Api.V1`, you must have path like this `controllers/api/v1/user_controller.ex`

Reply:
Should I also move other controllers to the v1 folder?

Reply:
Yes

Reply:
It's our fail